lib/config/config-validator.js

```diff
@@ -164,17 +173,23 @@ function validateEnvironment(environment, envContext, source = null) {
 /**
  * Validates a rules config object
  * @param {Object} rulesConfig The rules config object to validate.
- * @param {function(string): {create: Function}} ruleMapper A mapper function from strings to loaded rules
  * @param {string} source The name of the configuration source to report in any errors.
+ * @param {function(ruleId:string): Object} getAdditionalRule A map from strings to loaded rules
  * @returns {void}
  */
-function validateRules(rulesConfig, ruleMapper, source = null) {
+function validateRules(
+    rulesConfig,
+    source,
+    getAdditionalRule = Function.prototype
```

Comment:
For clarity, can we use a variable called `noop` here rather than `Function.prototype` directly?